tools/test_perl_style.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
@jmagne or @edewata - Could I get review on this?

In particular: I'm introducing a new directory, `tools`. Is there anything special I need to do with respect to the spec file? I verified that it doesn't show up in the built RPM but not that it didn't get added to the jar archive.